src/shaders/glslang-c.cpp

```diff
@@ -78,3 +78,16 @@ extern "C"
 const char* GLSLangGetObjectCode(const ShHandle handle) {
     return sh::GetObjectCode(handle).c_str();
 }
+
+using StrPairFunction = void (*)(void *, const char *, size_t, const char *, size_t);
+
+extern "C"
+void GLSLangIterUniformNameMapping(const ShHandle handle, StrPairFunction each, void *closure_each) {
+    for (auto& uniform : *sh::GetUniforms(handle)) {
```

Comment:
There's nothing that guarantees that GetUniforms returns non-null, right? Like, it's non-null because _by default_ you pass `SH_VARIABLES`. But `compile` is still public, so either null-check here, or make compile `pub(crate)`?